DayZ Epoch 1.0.6 compatibility

oiad opened the pull request:
This updates DZAI with the new DayZ Epoch 1.0.6 class names and variable names.

oiad committed:
DayZ Epoch 1.0.6 compatibility

oiad committed:
Missed DZ_Czech_Vest_Puch update

Thanks @TheZog

oiad commented:
Fixed, cheers!

looonytoonz commented:
is there a spot to download this with the updated code..? or do we have to download what is here and make the changes ourselves ?

oiad commented:
@looonytoonz This is the link to download from my repo https://github.com/oiad/DZAI/archive/master.zip

azhizhinov commented:
Hello there.

DannyBradley commented:
Any reason Im not seeing any vehicle patrols, including helis on namalsk ? I have updated the code changed at the top, do some vehicles have different ID's too ?

Commit:
Update 3. Install Instructions - Mission PBO Installation.txt

Commit:
Update 2. Install Instructions - dayz_server.pbo installation (PBOManager).txt

Commit:
Delete Readme.txt

Commit:
Update README.md

Commit:
Create 2.2.2.txt

Commit:
Update 2.2.2.txt

Release 2.2.2

Commit:
Update readme.txt

Commit:
Delete readme.txt

Commit:
Rename authentication_fix/$PBOPREFIX$ to $PBOPREFIX$

Commit:
Delete $PBOPREFIX$

Commit:
Delete $PREFIX$

Commit:
Delete PboPrefix.txt

Commit:
Rename authentication_fix/readme.txt to readme.txt

Commit:
Delete readme.txt

Commit:
Delete #readme.txt

Commit:
Update dzai_initclient.sqf

Commit:
Delete server_monitor_example.sqf

Commit:
Update and rename 3. Install Instructions - Mission PBO Installation.txt to Mission-Installation

Commit:
Delete 1. Install Instructions - dayz_server.pbo installation (cpbo).txt

Commit:
Rename Mission-Installation to Mission-Installation.md

Commit:
Update and rename 2. Install Instructions - dayz_server.pbo installation (PBOManager).txt to Server-Installation.md

Commit:
Update Server-Installation.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Create readme.md

Commit:
Update Mission-Installation.md

oiad committed:
Merge pull request #1 from DAmNRelentless/master

Reworked structure and install instruction, fixed client files

oiad committed:
Fix broken install line

Thanks @iben

AirwavesMan committed:
Update for Epoch 1.0.7

AirwavesMan committed:
Add coins on dead bodies

AirwavesMan committed:
Replace M4A3_CCO_EP1 with epoch version

AirwavesMan committed:
Fix sync error message

AirwavesMan committed:
Fix Epoch config got not detected when using Chernarus Winter

AirwavesMan committed:
Update installation links

AirwavesMan committed:
Fix Epoch config got not detected when using Taviana

ZzBombardierzZ committed:
Fix links calling wrong files

ZzBombardierzZ committed:
Fix error when calling initserver

`Error in expression <(isNil "_this") then {_this = []};
if ((count _this) > 0) then {

if ("readoverr>
  Error position: <count _this) > 0) then {

if ("readoverr>
  Error count: Type Object, expected Array,Config entry
File z\addons\dayz_server\DZAI\init\dzai_initserver.sqf, line 17`
The check on _this does not seem to work and is giving errors

AirwavesMan committed:
Merge pull request #3 from ZzBombardierzZ/master

Fix wrong links and errors

worldwidesorrow committed:
Update buildWeaponArrays.sqf

Update this function so it works with DayZ Mod 1.9.0 and DayZ Epoch 1.0.7+. Tested on Epoch 1.0.7.1.

worldwidesorrow committed:
Add Optional Attachments to AI Loadouts

Most of the weapons in the loot configs are base classes that can have attachments added. This adds the option of placing a random attachment in the AI gear to match the issued weapon.

oiad committed:
Merge pull request #4 from worldwidesorrow/master

Update buildWeaponArrays.sqf
